sys-apps/hwloc/hwloc-2.3.0.ebuild

```diff
@@ -40,6 +40,7 @@ BDEPEND="
 	virtual/pkgconfig
 "
 
+REQUIRED_USE="nvml? ( cuda )"
```

Comment:
I'm not familiar with this package, but I don't really see why the nvml flag needs to exist if it requires cuda, seems redundant. cuda also ensures the drivers are installed, although /if/ it directly links with nvidia-drivers libraries (haven't checked) it may need a := on drivers for safety.

Reply:
In a future, cuda may work on amd hardware